spec/javascripts/setupTests.js

```diff
@@ -3,4 +3,8 @@
 import Enzyme from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 
+import $ from 'jquery'
+global.$ = $
+global.jQuery = $
+
```

Comment:
Careful with this, it's overriding the mocks:
https://github.com/3scale/porta/blob/16e9f6eb97fff2b63a58f3d9bca7d3195f550581/spec/javascripts/__mocks__/global-mocks.js#L14
This is way I'd rather not use $ for tests, since it needs to be mocked for some React specs.